README.md

```diff
@@ -45,82 +64,81 @@ If you plan to develop *cwe_checker*, it is recommended to build it using the pr
 -   pytest >= 3.5.1 (for tests)
 -   SCons >= 3.0.5 (for tests)
 -   odoc >= 1.4 (for documentation)
--   [Rust](https://www.rust-lang.org) >= 1.44.1
-
-Just run `make all` to compile and register the plugin with BAP. You can run the test suite via `make test`. Documentation can be built via `make documentation`.
-
-### Local installation with Ghidra as backend ###
+-   [Rust](https://www.rust-lang.org) >= 1.49
 
-The Ghidra backend is still in early development, thus some checks are not yet available for it. To try it out, the following dependencies must be fulfilled:
--   [Rust](https://www.rust-lang.org) >= 1.44.1
--   [Ghidra](https://ghidra-sre.org/) >= 9.2. If you want to use an earlier version of Ghidra, you need to manually add the Java library `gson` to Ghidra: Download it from https://search.maven.org/artifact/com.google.code.gson/gson/2.8.6/jar and move it to the Ghidra plugin folder located at `$HOME/.ghidra/.ghidra_9.X.X_PUBLIC/plugins` (with the version number depending on your version of Ghidra).
-
-Run `make all GHIDRA_PATH=path/to/ghidra_folder` (with the correct path to the local Ghidra installation inserted) to compile and install the *cwe_checker*.
+Just run `make with_bap_backend` to compile and register the plugin with BAP.
 
 ## Usage ##
 
-The *cwe_checker* takes as input a binary file, runs several [checks](#checks) based on static analysis on the binary and then outputs a list of CWE warnings that have been found during the analysis.
```

Comment:
The *cwe_checker* takes a binary as input, ...?

Reply:
fixed